staging/src/k8s.io/apiserver/pkg/storage/cacher.go

```diff
@@ -526,11 +526,17 @@ func (c *Cacher) GuaranteedUpdate(
 	} else if exists {
 		currObj, copyErr := c.copier.Copy(elem.(*storeElement).Object)
 		if copyErr == nil {
-			return c.storage.GuaranteedUpdate(ctx, key, ptrToType, ignoreNotFound, preconditions, tryUpdate, currObj)
+			err := c.storage.GuaranteedUpdate(ctx, key, ptrToType, ignoreNotFound, preconditions, tryUpdate, currObj)
+			if !errors.IsConflict(err) {
+				// Return any error that isn't a conflict. Conflict errors should fall through to the
+				// no-suggestion update below.
+				return err
+			}
+		} else {
+			glog.Errorf("couldn't copy object: %v", copyErr)
```

Comment:
@smarterclayton all I did was move the previously existing line 👅 

@wojtek-t he wants `utilruntime.HandleError`